### PR TITLE
support iconTintColor

### PR DESCRIPTION
**Pull request by Bernix01**
Added support for template images and tint color

### CODE REVIEW

**Comment by k0nserv** [comment]
This looks good, can you add an entry to `CHANGELOG.md` too? CI is failing, I think we need to make some adjustments for different verisons of Swift

**Comment by Bernix01** [comment]
It's done

**Comment by Bernix01** [comment]
merge pls?

File: Sources/SkyFloatingLabelTextFieldWithIcon.swift [hunk 1]
```diff
@@ -218,7 +236,22 @@ open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
...
+        if self.iconType == .font {
+            updateIconLabelColor()
+        }
+        if self.iconType == .image {
+            updateImageViewTintColor()
+        }
+    }
+
+    fileprivate func updateImageViewTintColor() {
+        if !isEnabled {
```
Comment (k0nserv) [inline Sources/SkyFloatingLabelTextFieldWithIcon.swift]: Shouldn't this only be done if `templateImage` is `true`?

Comment (Bernix01) [reply Sources/SkyFloatingLabelTextFieldWithIcon.swift]: Change added. Although not doing so has no side effects

**Pull request merged**